cmd/metal-api/internal/service/ip-service.go

```diff
@@ -90,6 +90,17 @@ func (ir ipResource) webService() *restful.WebService {
 		DefaultReturns("Error", httperrors.HTTPErrorResponse{}))
 
 	ws.Route(ws.POST("/free/{id}").
+		To(editor(ir.freeIP)).
+		Operation("freeIPDeprecated").
+		Doc("frees an ip").
```

Comment:
`Consume("").` missing ?

Reply:
Right, probably makes it even safer to add this, too.